packages/frontend-shared/src/gql-components/topnav/TopNav.vue

```diff
@@ -357,3 +358,15 @@ const changelogLink = (version) => {
 }
 
 </script>
+
+<style>
+.docs-menu-flex-direction {
+  flex-direction: column;
+}
+
+@media(min-width: 768px) {
+  .docs-menu-flex-direction {
+    flex-direction: row;
+  }
+}
+</style>
```

Comment:
```suggestion
```
Remove in favor of Tailwind CSS

Reply:
AHHHH nevermind, I just re-read your GitHub description with the explanation on it not working